test/integration/fixtures/blocks/core__cover__gradient__deprecated-8.json

```diff
@@ -0,0 +1,33 @@
+[
+	{
+		"clientId": "_clientId_0",
+		"name": "core/cover",
+		"isValid": false,
```

Comment:
@ramonjd It was noticed [over here](https://github.com/WordPress/gutenberg/pull/38685#issuecomment-1035305924) that this block fixture results in an invalid block.

I'm not sure why that might be, it could be that something in the original html file was wrong (there was similar issue for a file block fixture - https://github.com/WordPress/gutenberg/pull/38725) or that there's an issue with the deprecation.

Any ideas?

Reply:
@talldan Thanks for the heads up. I've taken a look and it appears to be human error (mine).

The attribute `"isDark":false` should have a corresponding `is-light` classname on the container. 

Getting a PR together to fix this now.

Reply:
https://github.com/WordPress/gutenberg/pull/38728